js/app/nav.js

```diff
@@ -62,14 +62,15 @@ function cloneSecondaryNav() {
     if ($('body').hasClass('viewport-sm') && $('.js-nav-clone__list').find($navLink).length > 0) {
         // Remove from separate UL and add into primary
         $navLink.each(function() {
+            $(this).parent().addClass('hidden');
```

Comment:
I think it would be worth us using the hide function https://api.jquery.com/hide/ that handles this for us.